src/main.rs

```diff
@@ -69,9 +70,8 @@ fn run() -> Result<()> {
     let mut screen = Screen::new()?;
     let mut input = Input::new();
     input.run();
-    screen.init()?;
```

Comment:
I thought that was not needed. But now that you're raising concerns I think there's one case we're it will be needed: when starting the frontend without opening a file. In that case I think we do need to clear the screen.